dataeng/jobs/analytics/UserActivity.groovy

```diff
@@ -0,0 +1,72 @@
+                stringParam('EMR_USER_INFO', ' ', 'Additional github users with ssh access to cluster. Use blank space as default value.')
+                // END EMR cluster configuration
+                stringParam('TASK_USER', '$AUTOMATION_USER')
+                stringParam('TASKS_BRANCH', '$ANALYTICS_PIPELINE_RELEASE', 'Branch from the edx-analytics-pipeline repository. For tags use tags/[tag-name].')
+                stringParam('SECURE_REPO', extraVars.get('SECURE_REPO_URL'))
+                stringParam('SECURE_BRANCH', '$ANALYTICS_SECURE_RELEASE', 'Branch from the analytics-secure repository, where the configuration settings reside. For tags use tags/[tag-name]')
+                stringParam('SECURE_CONFIG', 'analytics-tasks/dev-edx.cfg', 'Config file from the analytics-secure repository for this job.')
+                stringParam('CONF_BRANCH', '$ANALYTICS_CONFIGURATION_RELEASE', 'Branch from the edx-analytics-configuration repository. For tags use tags/[tag-name].')
+                stringParam('NOTIFICATION_EMAILS', '$PAGER_NOTIFY', 'Space separated list of emails to notify in case of failure.')
+                stringParam('TO_DATE', 'today', 'The day after the last date to export data for.')
+                stringParam('NUM_TASK_CAPACITY', '8', 'Used to specify spot capacity of instance fleets core, also used to calculate the number of reducers by doubling this amount.')
+                booleanParam('TERMINATE', true, 'Boolean indicating whether to terminate the cluster on job completion or not.')
+                textParam('EXTRA_VARS', extraVars.get('AWS_EXTRA_VARS', ''))
+            }
+
+            multiscm{
+                git {
+                    remote {
+                        url('git@github.com:edx/edx-analytics-pipeline.git')
+                        branch('$TASKS_BRANCH')
+                        credentials('1')
+                    }
+                    extensions {
+                        pruneBranches()
+                        relativeTargetDirectory('analytics-tasks')
+                    }
+                }
+                git {
+                    remote {
+                        url('git@github.com:edx/edx-analytics-configuration.git')
+                        branch('$CONF_BRANCH')
+                        credentials('1')
+                    }
+                    extensions {
+                        pruneBranches()
+                        relativeTargetDirectory('analytics-configuration')
+                    }
+                }
+            }
+
+            wrappers {
+                timestamps()
+                sshAgent('1') // use credential's custom ID
+            }
+
+            steps {
+                shell(dslFactory.readFileFromWorkspace("dataeng/resources/user-activity.sh"))
+            }
+
+            publishers {
+                // Mark the build as 'unstable' if the text is found in 'console log'.
+                textFinder("\\[Traceback\\]", '', true, false, true)
```

Comment:
Is this a carry-over from exporter jobs, where we rarely fail the worker jobs and skip over content export failures?   We haven't really used the "unstable" state for our other jobs, but perhaps we should?   But would this mask our current use of Luigi error codes to cause jobs to fail?